### DISCUSSION TITLE
Horrid formatting when using PostCSS imports with Prettier

### DISCUSSION THREAD

**Discussion by deadcoder0904** [tailwindlabs/tailwindcss#2718] [Help] [answered]
I have some custom styles so I separated them by files.

My `index.css` looks like:

```css
@import url('./base.css') @import url('./grid.css') /* Start purging... */ @tailwind components;
/* Stop purging. */

/* Write your own custom component styles here */
.btn-blue {
	@apply px-4 py-2 font-bold text-white bg-blue-500 rounded;
}

/* Start purging... */
@tailwind utilities;
/* Stop purging. */

/* Your own custom utilities */
```

Note the horrid formatting above. I have installed related extensions but it always gives horrid formatting. Is there anyway to get it nicely formatted on Save? I'm using VSCode :)

**Comment by simonswiss** [comment]
Hello!

You can install a PostCSS language extension to vscode and set your index.css file to use the `PostCSS` language (in the bottom right, or via command palette). This should bring back some sanity to your formatting 👍

![image](https://user-images.githubusercontent.com/485747/97817176-c05c4a00-1cee-11eb-86c3-6125b50134db.png)

**Reply by deadcoder0904** [reply]
Welp, I already have both set.

**PostCSS Language Support** Extension installed:

![PostCSS Language Support Extension](https://user-images.githubusercontent.com/16436270/97830764-1c759d00-1cf4-11eb-87d5-2615a05d70b3.png)

PostCSS Language set for `.css` files:

![PostCSS Language set for `.css` files](https://user-images.githubusercontent.com/16436270/97830710-e7694a80-1cf3-11eb-8856-290d268769b4.png)

**Reply by deadcoder0904** [reply]
This is my `OUTPUT` tab after using Prettier. Even my config is pretty simple:

```bash
["INFO" - 10:36:06 AM] Detected local configuration (i.e. .prettierrc or .editorconfig), VS Code configuration will not be used
["INFO" - 10:36:06 AM] Using config file at '~/akshaykadam.com/prettier.config.js'
["INFO" - 10:36:06 AM] Prettier Options:
{
  "filepath": "~/akshaykadam.com/src/styles/index.css",
  "parser": "css",
  "semi": false,
  "singleQuote": true,
  "printWidth": 100,
  "tabWidth": 2,
  "useTabs": true,
  "trailingComma": "es5",
  "bracketSpacing": true
}
```

**Comment by deadcoder0904** [comment, marked as answer]
Solved it 🤦‍♂️

I forgot to use `;` in CSS as I don't use them in JS 😂

Horrible, lost so much time on this 🤣

**Reply by simonswiss** [reply]
Argh - makes sense now that you say this and I re-look at your code 😅

Glad you got it sorted and sorry I didn't pick up on that!
